types/node/crypto.d.ts

```diff
@@ -3531,7 +3531,7 @@ declare module "crypto" {
      * ```js
      * const { X509Certificate } = await import('node:crypto');
      *
-     * const x509 = new X509Certificate('{... pem encoded cert ...}');
+     * const x509 = new X509Certificate('{... pem/der encoded cert ...}');
```

Comment:
The docblocks are copied from the Node.js API documentation markdown. If you think that the example is unclear, I'd suggest submitting a change proposal upstream.